### PR TITLE
Change layout of build output directory

### LINKED ISSUE #301: Change Framework installation directory in Homebrew
I have compared some other CLI tools that are depending on some Mac OS framework. A good example is Carthage: Once you install it via Homebrew, you'll find following result in `/usr/local`:

```
$ l /usr/local/Frameworks/CarthageKit.framework/
total 16
lrwxr-xr-x  1 joplunien  admin   73 Jul 25 14:07 CarthageKit -> ../../Cellar/carthage/0.17.2/Frameworks/CarthageKit.framework/CarthageKit
lrwxr-xr-x  1 joplunien  admin   71 Jul 25 14:07 Resources -> ../../Cellar/carthage/0.17.2/Frameworks/CarthageKit.framework/Resources
drwxr-xr-x  4 joplunien  admin  136 Jul 25 14:07 Versions
```

It would be great if we could achieve the same thing for `fbsimctl`. Right now the framework is only available in `/usr/local/Cellar/fbsimctl/HEAD-b201a46/libexec`.

The use case is:

```swift
#!/usr/bin/env xcrun swift -F /usr/local/Cellar/fbsimctl/HEAD-b201a46/libexec

import FBSimulatorControl

let options = FBSimulatorManagementOptions.KillSpuriousSimulatorsOnFirstStart
let config = FBSimulatorControlConfiguration(deviceSetPath: nil, options: options)
let logger = FBControlCoreGlobalConfiguration.defaultLogger()
try FBSimulatorControlFrameworkLoader.loadPrivateFrameworks(logger)
let control = try FBSimulatorControl.withConfiguration(config, logger: logger)
let device = FBSimulatorConfiguration.iPhone5()
let simulator = try control.pool.allocateSimulatorWithConfiguration(device, options: [.Create, .Reuse, .EraseOnAllocate])
try simulator.interact.bootSimulator().perform()
```

`-F /usr/local/Cellar/fbsimctl/HEAD-b201a46/libexec` should actually be `-F /usr/local/Frameworks`

### PR DESCRIPTION
**Pull request by plu**
Fix #301

### CODE REVIEW

File: fbsimctl/fbsimctl.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -610,13 +610,15 @@
 			isa = XCBuildConfiguration;
 			baseConfigurationReference = AA7D2F5F1D33844000C7C6CF /* fbsimctl.xcconfig */;
 			buildSettings = {
+				LD_RUNPATH_SEARCH_PATHS = "@executable_path/../Frameworks $(inherited)";
```
Comment (lawrencelomax) [inline fbsimctl/fbsimctl.xcodeproj/project.pbxproj]: You should be able to put this in the `fbsimctl.xcconfig` instead of the Project itself.

**Comment by plu** [comment]
I'm experimenting with `install_name_tool`. Maybe I'll find a better solution, so setting `LD_RUNPATH_SEARCH_PATHS` might not be required at all.

**Comment by lawrencelomax** [comment]
Bear in mind that homebrew will 'relocate' Frameworks by using otool to rewrite the install path of Frameworks. You can verify this by switching on verbose logging when you brew install. 

If the Frameworks are installed to a global path, like you've been working towards this should work out.

**Comment by plu** [comment]
I'm giving up on `install_name_tool`. The use case from #301 is working though.

**Comment by lawrencelomax** [comment]
lgtm! Will wait for build results to come in and then land.

**Comment by plu** [comment]
Sounds good!

I really appreciate you taking time for all the reviews! Thank you :)!

**Comment by lawrencelomax** [comment]
Build looks good, the failing test will be green once this lands on master 

@facebook-github-bot shipit

**Comment by facebook-github-bot-9** [comment]
Thanks for importing.If you are an FB employee go to [Phabricator](https://phabricator.intern.facebook.com/D3728762) to review internal test results.

**Pull request closed without merging**